conda_smithy/feedstock_content/build-locally.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+#
+# This file has been generated by conda-smithy in order to build the recipe
+# locally.
+#
+import os
+import glob
+import subprocess
+from argparse import ArgumentParser
+
+
+def setup_environment(ns):
+    os.environ["CONFIG"] = ns.config
+    os.environ["UPLOAD_PACKAGES"] = "False"
+
+
+def run_docker_build(ns):
+    script = glob.glob(".*/run_docker_build.sh")[0]
+    subprocess.check_call(script)
+
+def verify_config(ns):
+    valid_configs = {os.path.basename(f)[:-5] for f in glob.glob(".ci_support/*.yaml")}
+    print(f"valid configs are {valid_configs}")
```

Comment:
Add a check for python 3.6+ somewhere?

Reply:
Won't it just fail to compile if it not Python 3.6?

Reply:
yeah for py<35 we'll just get a SyntaxError